pubsub_logging/pubsub_handler.py

```diff
@@ -74,8 +75,9 @@ def __init__(self, topic, capacity=DEFAULT_BATCH_NUM,
         self._publish_body = publish_body
         if client:
             self._client = client
-        else:  # pragma: NO COVER
+        else:
             self._client = get_pubsub_client()
+        create_topic(self._client, topic, retry)
```

Comment:
Hrm, I thought you had this code before, but decided to remove it for some reason. Do you remember that?